pkg/tecdsa/signing/protocol_test.go

```diff
@@ -1293,4 +1482,68 @@ func initializeTssRoundThreeMembersGroup(
 	}
 
 	return tssRoundThreeMembers, tssRoundTwoMessages, nil
+}
+
+func initializeTssRoundFourMembersGroup(
+	dishonestThreshold int,
+	groupSize int,
+) (
+	[]*tssRoundFourMember,
+	[]*tssRoundThreeMessage,
+	error,
+) {
+	var tssRoundFourMembers []*tssRoundFourMember
+	var tssRoundThreeMessages []*tssRoundThreeMessage
+
+	tssRoundThreeMembers, tssRoundTwoMessages, err :=
+		initializeTssRoundThreeMembersGroup(
+			dishonestThreshold,
+			groupSize,
+		)
+	if err != nil {
+		return nil, nil, fmt.Errorf(
+			"cannot generate TSS round three members group: [%v]",
+			err,
+		)
+	}
+
+	for _, member := range tssRoundThreeMembers {
+		var receivedTssRoundTwoMessages []*tssRoundTwoMessage
+		for _, tssRoundTwoMessage := range tssRoundTwoMessages {
+			if tssRoundTwoMessage.senderID != member.id {
+				receivedTssRoundTwoMessages = append(
+					receivedTssRoundTwoMessages,
+					tssRoundTwoMessage,
+				)
+			}
+		}
+
+		ctx, cancelCtx := context.WithTimeout(
+			context.Background(),
+			10*time.Second,
+		)
+
+		tssRoundThreeMessage, err := member.tssRoundThree(
+			ctx,
+			receivedTssRoundTwoMessages,
+		)
+		if err != nil {
+			cancelCtx()
+			return nil, nil, fmt.Errorf(
+				"cannot do TSS round three for member [%v]: [%v]",
+				member.id,
+				err,
+			)
+		}
+
+		tssRoundFourMembers = append(
+			tssRoundFourMembers,
+			member.initializeTssRoundFour(),
+		)
+		tssRoundThreeMessages = append(tssRoundThreeMessages, tssRoundThreeMessage)
+
+		cancelCtx()
+	}
+
+	return tssRoundFourMembers, tssRoundThreeMessages, nil
 }
```

Comment:
It seems the files is not formatted with `go fmt`.

Reply:
Fixed after https://github.com/keep-network/keep-core/pull/3252 was merged to main.